### PR TITLE
Fix potential NPE

### PR DESCRIPTION
**Pull request by mmorhun**
Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

### What does this PR do?
Prevents potential NPE.
Fixes local debug command

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18474

### CODE REVIEW

**Review by AndrienkoAleksandr — APPROVED** [review]

**Comment by AndrienkoAleksandr** [comment]
@mmorhun I think check-nightly bundle force you to update bundles, because was released new ubi image... @tolusha What do you think about such case? Should we improve nigthly check to avoid force bundle update for this case?

File: pkg/deploy/server/che_configmap.go [hunk 1]
```diff
@@ -282,6 +282,15 @@ func GetCheConfigMapData(deployContext *deploy.DeployContext) (cheEnv map[string
 			if err != nil {
 				return nil, err
 			}
+			if cheTLSSecret.Data == nil {
```
Comment (tolusha) [inline pkg/deploy/server/che_configmap.go:285]:
```suggestion
			if cheTLSSecret == nil {
```

File: pkg/deploy/server/che_configmap.go [hunk 2]
```diff
@@ -282,6 +282,15 @@ func GetCheConfigMapData(deployContext *deploy.DeployContext) (cheEnv map[string
...
+				return nil, fmt.Errorf("%s secret has no data", deployContext.CheCluster.Spec.K8s.TlsSecretName)
```
Comment (tolusha) [inline pkg/deploy/server/che_configmap.go:286]:
```suggestion
				return nil, fmt.Errorf("%s secret not found", deployContext.CheCluster.Spec.K8s.TlsSecretName)
```

**Comment by tolusha** [comment]
@AndrienkoAleksandr 
No, this check is fine.

**Comment by mmorhun** [comment]
@tolusha we are going to ignore it, right?

File: pkg/deploy/server/che_configmap.go [hunk 3]
```diff
@@ -282,6 +282,15 @@ func GetCheConfigMapData(deployContext *deploy.DeployContext) (cheEnv map[string
...
+			}
+			if _, exists := cheTLSSecret.Data["tls.key"]; !exists {
+				return nil, fmt.Errorf("%s secret has no tls.key field in data", deployContext.CheCluster.Spec.K8s.TlsSecretName)
```
Comment (tolusha) [inline pkg/deploy/server/che_configmap.go:289]:
```suggestion
				return nil, fmt.Errorf("%s secret has no 'tls.key' key in data", deployContext.CheCluster.Spec.K8s.TlsSecretName)
```

File: pkg/deploy/server/che_configmap.go [hunk 4]
```diff
@@ -282,6 +282,15 @@ func GetCheConfigMapData(deployContext *deploy.DeployContext) (cheEnv map[string
...
+			}
+			if _, exists := cheTLSSecret.Data["tls.crt"]; !exists {
+				return nil, fmt.Errorf("%s secret has no tls.crt field in data", deployContext.CheCluster.Spec.K8s.TlsSecretName)
```
Comment (tolusha) [inline pkg/deploy/server/che_configmap.go:292]:
```suggestion
				return nil, fmt.Errorf("%s secret has no 'tls.crt' key in data", deployContext.CheCluster.Spec.K8s.TlsSecretName)
```

**Review by tolusha — APPROVED** [review]

**Pull request merged**